lib/ApiRequestor.php

```diff
@@ -100,6 +102,8 @@ public function handleApiError($rbody, $rcode, $rheaders, $resp)
                 throw new Error\Authentication($msg, $rcode, $rbody, $resp, $rheaders);
             case 402:
                 throw new Error\Card($msg, $param, $code, $rcode, $rbody, $resp, $rheaders);
+            case 429:
```

Comment:
Can we add a comment on the 400 rate limit that identifies that it's backwards compat for API versions older than X